packages/edit-site/src/components/sidebar-navigation-screen-page/index.js

```diff
@@ -18,6 +33,78 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import { unlock } from '../../private-apis';
 import { store as editSiteStore } from '../../store';
 import SidebarButton from '../sidebar-button';
+import SidebarNavigationSubtitle from '../sidebar-navigation-subtitle';
+import SidebarDetails from '../sidebar-navigation-data-list';
+import DataListItem from '../sidebar-navigation-data-list/data-list-item';
+import StatusLabel from './status-label';
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+function getPageDetails( page ) {
+	if ( ! page ) {
+		return [];
+	}
+	const details = [];
+
+	if ( page?.status ) {
```

Comment:
Can we have pages without `status` or `slug`?

Reply:
In my experience, no, but I wouldn't expect it hurts to be defensive. Happy to remove these checks since we're already checking if the record is not `undefined` (which it is until the selector resolves)